homeassistant/components/unifi/sensor.py

```diff
@@ -178,7 +178,7 @@ class UnifiSensorEntityDescription(
         value_fn=lambda _, obj: obj.poe_power if obj.poe_mode != "off" else "0",
     ),
     UnifiSensorEntityDescription[Clients, Client](
-        key="Uptime sensor",
+        key="Client uptime",
```

Comment:
Only double-checking. This renaming doesn't have any functional consequences, right?

The rest is fine, and I'm happy to merge :)

Reply:
Correct. It's a descriptive text only